src/Logging/src/DynamicLogger/ConsoleLoggerOptionsSetup.cs

```diff
@@ -1,17 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the Apache 2.0 License.
-// See the LICENSE file in the project root for more information.
-
-using Microsoft.Extensions.Logging.Configuration;
-using Microsoft.Extensions.Logging.Console;
-using Microsoft.Extensions.Options;
-
-namespace Steeltoe.Logging.DynamicLogger;
-
-internal sealed class ConsoleLoggerOptionsSetup : ConfigureFromConfigurationOptions<ConsoleLoggerOptions>
```

Comment:
I don't know if this impacts how we are using this but not all of `ConsoleLoggerOptions`[link](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consoleloggeroptions?view=dotnet-plat-ext-7.0) has been deprecated - just most of it. Do the remaining two have a bearing on how our dynamic logger is  configured?

Reply:
[MaxQueueLength](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consoleloggeroptions.maxqueuelength?view=dotnet-plat-ext-7.0#microsoft-extensions-logging-console-consoleloggeroptions-maxqueuelength) and [QueueFullMode](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consoleloggeroptions.queuefullmode?view=dotnet-plat-ext-7.0#microsoft-extensions-logging-console-consoleloggeroptions-queuefullmode) were added in .NET 7, [LogToStandardErrorThreshold](https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consoleloggeroptions.logtostandarderrorthreshold?view=dotnet-plat-ext-7.0#microsoft-extensions-logging-console-consoleloggeroptions-logtostandarderrorthreshold) was introduced in .NET Core 3.

These settings are not relevant for dynamic logging, however they are consumed by its wrapped logger. In that case, I don't think we need to do anything to have the built-in console logger consume these settings.

Reply:
I've manually tested that setting `Logging:Console:LogToStandardErrorThreshold = "Information"` in appsettings.json works when dynamic logging is used. Similarly to scopes, this is pretty hard to unit-test.

Reply:
Added a test that uses the obsolete boolean `DisableColors`, so this should be covered now.